lib/openhab/dsl/things/builder.rb

```diff
@@ -244,16 +275,26 @@ def thing_type
 
       # The BridgeBuilder DSL allows you to customize a thing
       class BridgeBuilder < ThingBuilder
+        # @!visibility private
+        attr_accessor :caller
```

Comment:
can we think of a better name for this? parent_builder or something? caller is just a method on Kernel to get the current call stack, which can be confusing if you overwrite it

Reply:
I've changed it to `parent_builder`